lib/robots/dor_repo/base.rb

```diff
@@ -5,6 +5,11 @@ module DorRepo
     class Base
       include LyberCore::Robot
 
+      def initialize(...)
+        super
+        Cocina::Models::Mapping::Purl.base_url = Settings.purl_url
```

Comment:
Can this be in the DescriptiveMetadata robot or in config/boot.rb ?

Reply:
Ahh, I did not know about `boot.rb`. Fixed.